src/main/java/uk/gov/companieshouse/web/pps/util/PenaltyUtilsService.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.companieshouse.web.pps.util;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.text.DecimalFormat;
+
+@Service
+public class PenaltyUtilsService {
+
+    public final String viewPenaltiesLateFilingReason;
+
+    public PenaltyUtilsService(@Value("${penalty.view-penalties-late-filing-reason}") String viewPenaltiesLateFilingReason){
+        this.viewPenaltiesLateFilingReason = viewPenaltiesLateFilingReason;
+    }
+
+    public String getPenaltyReason() {
```

Comment:
Should this method be renamed to be `getViewPenaltiesLateFilingReason` to match the property name above?

Reply:
Fixed in latest commit